pom.xml

```diff
@@ -58,24 +58,25 @@
     <properties>
         <otp.serialization.version.id>76</otp.serialization.version.id>
         <!-- Lib versions - keep list sorted on property name -->
-        <geotools.version>27.0</geotools.version>
-        <google.dagger.version>2.42</google.dagger.version>
+        <geotools.version>27.1</geotools.version>
+        <google.dagger.version>2.44</google.dagger.version>
+        <!-- Cannot update Jackson until this is merged: https://github.com/OneBusAway/onebusaway-gtfs-modules/pull/206 -->
         <jackson.version>2.13.3</jackson.version>
         <jersey.version>2.34</jersey.version>
-        <junit.version>5.8.2</junit.version>
-        <micrometer.version>1.8.10</micrometer.version>
+        <junit.version>5.9.1</junit.version>
+        <micrometer.version>1.9.5</micrometer.version>
         <netcdf4.version>5.5.2</netcdf4.version>
         <netty.version>4.1.74.Final</netty.version>
-        <lucene.version>9.1.0</lucene.version>
-        <graphql.version>18.3</graphql.version>
+        <lucene.version>9.4.1</lucene.version>
+        <graphql.version>19.2</graphql.version>
```

Comment:
We might need to fix `MicrometerGraphQLInstrumentation` due to https://github.com/graphql-java/graphql-java/pull/2769#discussion_r931658926